_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
> It might be worth talking about the benefits of being able to customize the application lifecycle. It might also be worth contrasting Service Workers with traditional ways of maintaining freshness, and explaining how Service Workers are designed to improve the reliability of application updates (perhaps by having the ACID properties, unsure). But freshness itself isn't a selling point of PWAs specifically.

Myk, I started writing the response before reading this comment. I think we agree on that. :D